service/provider/src/main/java/gov/va/vro/service/provider/camel/MasIntegrationProcessors.java

```diff
@@ -0,0 +1,77 @@
+package gov.va.vro.service.provider.camel;
+
+import gov.va.vro.camel.FunctionProcessor;
+import gov.va.vro.model.HealthDataAssessment;
+import gov.va.vro.model.VeteranInfo;
+import gov.va.vro.model.mas.MasAutomatedClaimPayload;
+import gov.va.vro.service.provider.mas.MasException;
+import gov.va.vro.service.provider.mas.service.MasCollectionService;
+import gov.va.vro.service.provider.mas.service.MasTransferObject;
+import gov.va.vro.service.spi.model.Claim;
+import gov.va.vro.service.spi.model.GeneratePdfPayload;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+import java.util.List;
+import java.util.function.Function;
+
+/** Helper processors for Mas Integration */
+@Slf4j
+public class MasIntegrationProcessors {
```

Comment:
extracted this helper class to reduce the size of MasIntegrationRoutes and hopefully make it more readable